src/encoded/tests/test_upgrade_target.py

```diff
@@ -70,3 +81,10 @@ def test_target_upgrade_status_5_6(upgrader, target_5):
         'target', target_5, current_version='5', target_version='6')
     assert value['schema_version'] == '6'
     assert value['status'] == 'current'
+
+
+def test_target_upgrade_status_6_7(upgrader, target_6):
```

Comment:
You could rename this function to something like `test_target_upgrade_remove_histone_modification_6_7`.

Reply:
renamed the function & pushed change